.travis.yml

```
@@ -33,18 +41,37 @@ script:
 jobs:
   fast_finish: true
   allow_failures:
+    - php: 7.3
     - php: nightly
   include:
+    - php: 7
+      env: COMPOSER_ARGS="--prefer-lowest"
+    - php: 7
+    - php: 7.1
+      env: COMPOSER_ARGS="--prefer-lowest"
+    - php: 7.1
+    - php: 7.2
+      env: COMPOSER_ARGS="--prefer-lowest"
+    - php: 7.2
+    - php: 7.3
+      env: COMPOSER_ARGS="--prefer-lowest"
+    - php: 7.3
+    - php: nightly
+      env: COMPOSER_ARGS="--prefer-lowest"
```

for 7.3 and nightly we should ignore the platform requirements so the tests will run on these versions (otherwise php-cs-fixer keeps composer from installing dependencies).

```suggestion
      env: COMPOSER_ARGS="--prefer-lowest --ignore-platform-reqs"
```

---

Ah, gotcha. I saw that in one of your changes, but thought I'd try without since I wasn't sure the reason.